features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/default/js/logic/recent-assets.js

```diff
@@ -39,7 +39,7 @@ $(function () {
             type: 'GET',
             success: function (response) {
 
-                var respObj=JSON.parse(response);
+                var respObj = response;
```

Comment:
removed redundant variable 'respObj' in commit d4a30b7ece56020f5f8934938f6e361631d86c30 of PR #240